index.js

```diff
@@ -8,7 +8,9 @@ const kWs = Symbol('ws-socket')
 const kWsHead = Symbol('ws-head')
 
 function fastifyWebsocket (fastify, opts, next) {
-  let errorHandler = defaultErrorHandler
+  fastify.decorateRequest('ws', true)
```

Comment:
Just for safety I think it'd make more sense for this to be null by default so that if something changes later with the request hook, it's clear if it has or hasn't been set yet, instead of maybe being a lie (`true`), you know?